src/plone/restapi/serializer/dxcontent.py

```diff
@@ -55,11 +56,13 @@ def __call__(self, version=None, include_items=True):
         parent_summary = getMultiAdapter(
             (parent, self.request), ISerializeToJsonSummary
         )()
+
         result = {
             # '@context': 'http://www.w3.org/ns/hydra/context.jsonld',
             "@id": obj.absolute_url(),
             "id": obj.id,
             "@type": obj.portal_type,
+            "type": get_portal_type(obj.portal_type, self.request),
```

Comment:
`@type` vs `type` is at a maximum confusing. I would propose to use something like `type_title` or `type_name`.

Reply:
Changed type to type_name